src/Models/Product.php

```diff
@@ -51,6 +52,11 @@ public function getUrlAttribute()
         return route('products.show', ['product' => $this->attributes['slug']]);
     }
 
+    public function getVariantCountAttribute()
+    {
+        return sprintf('%s %s', $count = $this->variants->count(), Str::plural('variant', $count));
```

Comment:
I don't really putting this code inside the model as an attribute as it's not being called anywhere else apart from in a single view. 

I would prefer if it was just done inline.

Reply:
What if the user would like to show this on the front-end somewhere, maybe on the product list page?

Reply:
Hmm, yeah that seems like a plausible use case. We might want to add it to the `appends` list if we want it to be available in templates.

Reply:
Sure, I had it on the appends at first then removed it lol. Will add back now.